.eslintignore

```diff
@@ -0,0 +1,3 @@
+rollup.config.js
+dist/**/*.js
+node_modules/**/*.js
```

Comment:
newline? I'm seeing this a lot lately - does VS Code strip EOF newlines by default or something?